### PR TITLE
Add iOS 9.3 SDK

### PR DESCRIPTION
**Pull request by urbaniak**

### CODE REVIEW

**Comment by stefanalund** [comment]
Thanks :+1:

File: config/ios.config [hunk 1]
```diff
@@ -29,7 +29,8 @@ _toolchain_root = '%s/usr/bin' % iphone_platform
 
 # Find the installed SDK
 _sdk_version = None
-for d,v in [(DistroVersion.IOS_9_2, '9.2'),
+for d,v in [(DistroVersion.IOS_9_2, '9.3'),
+            (DistroVersion.IOS_9_2, '9.2'),
```
Comment (sdroege) [inline config/ios.config:33]: This commit is obviously not complete: you also need to add an enum here

Comment (stefanalund) [inline config/ios.config:33]: Ouch, I missed that as well :smirk:

**Comment by sdroege** [comment]
The complete commit for adding iOS SDK 9.3 support can be found here: https://cgit.freedesktop.org/gstreamer/cerbero/commit/?id=5bd439644448be99c95002ad5ae7f714220b0a5b

**Comment by superdump** [comment]
I've merged in @sdroege 's commit as well as everything else from cerbero upstream master.

**Pull request closed without merging**